docs/project/reward-algorithm.mdx

```diff
@@ -5,52 +5,48 @@ sidebar_label: Reward Algorithm
 slug: /reward-algorithm
 ---
 
-# Reward Algorithm
-
 Rewarding the members of our network for sharing their local weather data is a crucial part of our system (**currently on testnet**).
 
-Our **Reward Algorithm** is a unique, fair and transparent algorithm that will handle how rewards are being distributed to the network members.
+Our **Reward Algorithm** is a unique, fair, and transparent algorithm that will handle how rewards are being distributed to the network members.
```

Comment:
I really like that you are using the [Oxford Comma](https://en.wikipedia.org/wiki/Serial_comma) notation throughout the doc! Hated by many, but I love it, as it ads clarity when needed. However, let's not overuse it when not necessary. For example here I believe we could skip it. :-)

Reply:
I will remove it, but the serial comma is preferred (often mandatory) in American academic and formal writing. I did not realize that other regions regard the serial comma as a preference, rather than a standard, until I read through your referenced article. 